calculation.js

```diff
@@ -0,0 +1,10 @@
+
+function strictEqual(number1, number2) {
+  if (number1 == number2 && typeof number1 == typeof number2) {
+    return true
+  } else {
+    return false
+  }
+}
```

Comment:
This is a great solution. Just something in the future to possibly make your life easier; if there is only one if-statement in the function, you can make it a one liner, as follows:  
```
function strictEqual(val1, val2) {
  return val1 == val2 && typeof val1 == typeof val2
}
```